e2e/tests/api_spec_claim_test/api_spec_1v1_lip_lip_directions_order_test.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars */
+const config = require('../../config.js');
+const { assert } = require('chai');;
+let civilCaseReference;
+
+Feature('Create Lip v Lip claim -  Default Judgment');
```

Comment:
test tag is needed, @api-nightly or @api-cui ?

Reply:
we will also need feature toggle condition to decide whether or not to run these tests on different env. in AAT we should not run this

Reply:
added tag api-cui